code/API_definitions/carrier_billing_payment.yaml

```diff
@@ -774,10 +817,15 @@ components:
       required:
         - code
         - message
+        - status
       properties:
         code:
           type: string
           description: Code given to this error
+        status:
+          type: string

```

Comment:
```suggestion
          type: integer

```

Reply:
Has been agreed in commonalities.
I am going to do the same in the checkout proposal

https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-design-guidelines.md#6-error-responses